docs/configuration/listeners/ssl.rst

```diff
@@ -43,12 +43,26 @@ alt_alpn_protocols
 ca_cert_file
   *(optional, string)* A file containing certificate authority certificates to use in verifying
   a presented client side certificate. If not specified and a client certificate is presented it
-  will not be verified.
+  will not be verified. By default, client certificate is optional, unless one of the additional
+  options (
+  :ref:`require_client_certificate <config_listener_ssl_context_require_client_certificate>`,
```

Comment:
So if i read this correctly, i'll have to specify `require_client_certificate` and `ca_cert_file` to get mutual authentication?